lib/sandboxed_renderer/init.js

```diff
@@ -1,9 +1,9 @@
 /* eslint no-eval: "off" */
 /* global binding, Buffer */
 const events = require('events')
-const electron = require('electron')
 
 process.atomBinding = require('../common/atom-binding-setup')(binding.get, 'renderer')
+const electron = require('electron')
```

Comment:
Maybe there should a comment explaining that the order of these two statement matters?

Reply:
done